apps/daimo-mobile/src/view/screen/YourInvitesScreen.tsx

```diff
@@ -0,0 +1,97 @@
+import { EAccount, timeAgo } from "@daimo/common";
+import {
+  ScrollView,
+  StyleSheet,
+  TouchableHighlight,
+  TouchableOpacity,
+  View,
+} from "react-native";
+
+import { navToAccountPage, useNav } from "../../common/nav";
+import { Account } from "../../model/account";
+import { ContactBubble } from "../shared/ContactBubble";
+import { ScreenHeader } from "../shared/ScreenHeader";
+import { color, ss, touchHighlightUnderlay } from "../shared/style";
+import { TextBody } from "../shared/text";
+import { useWithAccount } from "../shared/withAccount";
+
+export function YourInvitesScreen() {
+  const Inner = useWithAccount(YourInvitesScreenInner);
+  return <Inner />;
+}
+
+function YourInvitesScreenInner({ account }: { account: Account }) {
+  const nav = useNav();
+
+  const invitees = account.invitees;
+
+  return (
+    <View style={ss.container.screen}>
+      <ScreenHeader title="Your Invites" onBack={nav.goBack} />
+      <ScrollView style={styles.list}>
+        {invitees.map((invitee) => (
+          <InviteeRow key={invitee.addr} invitee={invitee} />
+        ))}
+      </ScrollView>
+    </View>
+  );
+}
+
+function InviteeRow({ invitee }: { invitee: EAccount }) {
+  const nav = useNav();
+
+  return (
+    <View style={{ marginHorizontal: 16 }}>
+      <TouchableHighlight
+        onPress={() => navToAccountPage(invitee, nav)}
+        {...touchHighlightUnderlay.subtle}
+        style={styles.rowUnderlayWrap}
+      >
+        <View style={styles.inviteeRow}>
+          <View style={styles.inviteeRowLeft}>
+            <TouchableOpacity onPress={() => navToAccountPage(invitee, nav)}>
+              <ContactBubble contact={{ type: "eAcc", ...invitee }} size={36} />
```

Comment:
if the whole row already navs to the account page, then i don't think that the contact bubble needs to be individually tappable

Reply:
done